src/python/foglamp/__main__.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+import foglamp.core.core_daemon
+
+
+if __name__ == "__main__":
+    foglamp.core.core_daemon.main()
```

Comment:
I am following this convention:

main() takes not arguments and processes command-line arguments
start() only works with input parameters